tests/Unit/Bridge/Doctrine/Repositories/Lifecycle/RequestRepositoryTest.php

```diff
@@ -128,12 +129,45 @@ public function testGetFailedRequestsOmitsSucceededRequests(): void
         self::assertEquals($expected, $results);
     }
 
+    /**
+     * Tests that getFillIterable returns expected data.
+     *
+     * @return void
+     *
+     * @throws \Doctrine\ORM\ORMException
+     * @throws \EoneoPay\Utils\Exceptions\InvalidDateTimeStringException
+     * @throws \ReflectionException
+     */
+    public function testGetFillIterable(): void
+    {
+        $requestData = new WebhookRequestData($this->getEntityManager());
+        $requestData
+            ->createRequest(new DateTime('2019-10-10 12:00:00'), 1)
+            ->createRequest(new DateTime('2019-10-11 12:00:00'), 2)
+            ->createRequest(new DateTime('2019-10-12 12:00:00'), 3)
+            ->build();
+
+        $expectedRequests = $requestData->getRequests();
+
+        $repository = $this->getRepository();
+
+        $iterable = $repository->getFillIterable();
+        /** @noinspection PhpParamsInspection Phpstorm is wrong. iterator is acceptable. */
```

Comment:
Is there a better way we can handle this so we don't have to use `@noinspection` any where that we use `getFillIterable()`?

Reply:
Unfortunately not, `iterable` is fine unless we set return type for `getFillIterable()` to `Traversable`

Reply:
You wont need to use @noinspection, getFillIterable is only used by the search package internally.

Additionally, the suppression is only required because the typehints for `iterator_to_array` are wrong, not because iterable is wrong.

Traversable is not the correct typehint.

Reply:
phpstan error
```
 ------ --------------------------------------------------------------------------------------------------------------------------------- 
  Line   tests/Unit/Bridge/Doctrine/Repositories/Lifecycle/RequestRepositoryTest.php                                                      
 ------ --------------------------------------------------------------------------------------------------------------------------------- 
  155    Parameter #1 $it of function iterator_to_array expects Traversable, iterable<EoneoPay\Webhooks\Bridge\Doctrine\Entities\Entity>  
         given.                                                                                                                           
 ------ --------------------------------------------------------------------------------------------------------------------------------- 

```